src/tests/testData.ts

```diff
@@ -13,6 +13,9 @@ export const testRecords: AirtableRecord[] = [
         denominator: 2000,
```

Comment:
@ewanmay @brett-stephen are we even using this test data anymore lol. Let's make a task to either incorporate these into proper tests or get rid of them if we don't have any immediate plans to do this. Seems like a waste of time to be maintaining this.